docs/onboard-vendors-to-techpass.md

```diff
@@ -0,0 +1,100 @@
+# Onboarding Vendors
+This section guides how vendors or contractors working on Gov ICT Projects can get a TechPass account and onboard in to it.
+
+## How to get a TechPass account for vendors or contractors?
+
+If you are a vendor or contractor who does not have an active TechPass account, request one from your project manager or reporting officer. Make sure you provide your official email address, handphone number and your project name in your request. Your project manager or reporting officer in turn will work with the engaging government agency to set up a TechPass account for you.
+
+?>
+  You can use this TechPass account for projects hosted by all government agencies.<br><br>
```

Comment:
You can use TechPass account to access SGTS products and services

Reply:
Done